packages/webview_flutter/webview_flutter_wkwebview/example/integration_test/webview_flutter_test.dart

```diff
@@ -861,8 +861,8 @@ Future<void> main() async {
 
   group('NavigationDelegate', () {
     const String blankPage = '<!DOCTYPE html><head></head><body></body></html>';
-    final String blankPageEncoded = 'data:text/html;charset=utf-8;base64,' +
-        base64Encode(const Utf8Encoder().convert(blankPage));
+    final String blankPageEncoded = 'data:text/html;charset=utf-8;base64,'
+        '${base64Encode(const Utf8Encoder().convert(blankPage))}';
```

Comment:
Is this some form of "avoid concatenation" of strings lint?

Reply:
Yep, it's https://dart-lang.github.io/linter/lints/prefer_interpolation_to_compose_strings.html

I think occasionally it's slightly worse, but not worth making exceptions for vs just switching